src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -3,15 +3,105 @@
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class GildedRoseTest {
 
     @Test
-    void foo() {
+    void qualityIsNeverNegative() {
         Item[] items = new Item[] { new Item("foo", 0, 0) };
         GildedRose app = new GildedRose(items);
         app.updateQuality();
-        assertEquals("fixme", app.items[0].name);
+        assertTrue(app.items[0].quality >= 0);
+    }
+
+    @Test
+    void qualityIsNeverMoreThan50() {
+        Item[] items = new Item[] { new Item("Aged Brie", 0, 50) };
```

Comment:
What happens if you do:
```Java
Item[] items = new Item[] { new Item("Aged Brie", 0, 100) };
```

Reply:
It looks like the previous developers did not add any safeguards around this use case, if any item is added to the inventory with a quality value over 50, the system will allow the items addition but its quality will never change.

So in this case, Aged Brie will remain with a quality of 100 no matter how many days pass.

We have added a bug ticket #13 to keep track of this issue and we will go over the requirements of this during the next IPM.